figures/scripts/fig3-oncoprint-landscape.R

```diff
@@ -0,0 +1,177 @@
+# Oncoprint Landscape Figure
+#
+# 2020
+# Chante Bethell for ALSF - CCDL
+#
+# This script is intended to run steps needed to create Figure 3.
+
+#### Set Up --------------------------------------------------------------------
+
+# Install maftools
+if (!("maftools" %in% installed.packages())) {
+  install.packages("BiocManager")
+  BiocManager::install("maftools")
+}
+library(maftools)
+
+# Load in patchwork for assembling the final multipanel figure
+library(patchwork)
+
+# Get `magrittr` pipe
+`%>%` <- dplyr::`%>%`
+
+#### Directories and Files -----------------------------------------------------
+
+# Detect the ".git" folder -- this will in the project root directory.
+# Use this as the root directory to ensure proper execution, no matter where
+# it is called from.
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+
+# Path to the data obtained via `00-map-to-sample_id.R`
+scratch_oncoprint_dir <- file.path(root_dir, "scratch", "oncoprint_files")
```

Comment:
Just checking, does this directory get all its necessary files from lines 53 - 76 of `generate_figures.sh`?

Reply:
That is correct.